apps/src/music/blockly/blocks/simple2.js

```diff
@@ -173,6 +177,42 @@ export const playRestAtCurrentLocationSimple2 = {
     `
 };
 
+export const setEffectAtCurrentLocationSimple2 = {
+  definition: {
+    type: BlockTypes.SET_EFFECT_AT_CURRENT_LOCATION_SIMPLE2,
+    message0: 'set %1 to %2',
+    args0: [
+      {
+        type: 'field_dropdown',
+        name: 'EFFECT',
```

Comment:
minor: create constant for `'EFFECT'` so it can be reused wherever it's accessed? We're doing the same for other similar field names

Reply:
Will do.